app/locales/en.json

```diff
@@ -33,9 +33,16 @@
     "home_no_contact_subtext": "Based on available data you haven’t been near anyone reported positive for COVID-19.",
     "home_unknown_header": "Unknown",
     "home_unknown_subtext": "We can’t tell if you’re at risk unless you enable the app to access your location.",
-    "import_step_1": "1. Login to your Google Account and Download your Location History",
-    "import_step_2": "2. After downloaded, open this screen again. The data will import automatically.",
-    "import_title": "Import Locations",
+    "import_title":"Import Locations",
+    "import_step_1":"Adding location data from Google will give you a head start on building your recent locations.",
+    "import_step_2":"Before you can import, you must first 'Take out' your location data from Google.",
+    "import_step_3":"Visit Google Takeout and export your Location History using following settings: \n1. Delivery method: \"Google Drive\" \n2. Frequency: \"Export once\" \n3. File type & size: \".zip\" and \"1GB\" \n4. Google sends an email when the export is ready \n5. Return here to import the archive",
```

Comment:
Delivery method is "Add to Drive", not Google Drive

<img width="309" alt="Screen Shot 2020-04-10 at 1 51 59 PM" src="https://user-images.githubusercontent.com/702990/79022293-746b6d00-7b32-11ea-831b-ef85b1b96f6b.png">